spacy/cli/project.py

```diff
@@ -236,13 +239,16 @@ def project_clone(
     # We're using Git and sparse checkout to only clone the files we need
     with make_tempdir() as tmp_dir:
         cmd = f"git clone {repo} {tmp_dir} --no-checkout --depth 1 --config core.sparseCheckout=true"
-        run_command(shlex.split(cmd))
+        try:
+            run_command(shlex.split(cmd, posix=not is_windows))
```

Comment:
Maybe it's worth it to move the `shlex.split` call out to a `util` function like `split_command`? This way, we can ensure that the `posix` flag is always set correctly (because it's easy to miss).

(I initially considered making the `run_command` helper take both a `List[str]` and `str` and then automatically split if the input is a string. It would make some things a bit cleaner, but on the other hand, I'm typically trying to avoid writing functions that take different input types.)